packages/devextreme/js/ui/date_range_box.d.ts

```diff
@@ -288,10 +298,11 @@ export type Properties = Omit<DateBoxBaseOptions<dxDateRangeBox>, 'inputAttr' |
     startDateText?: string;
     /**
      * @docid dxDateRangeBoxOptions.value
+     * @type Array<Date|number|string|null>
      * @default [null,null]
      * @public
      */
-    value?: Array<Date | number | string>;
+    value?: DateLike[];
```

Comment:
NOTE: It would be better to specify type like `[DateLike, DateLike]` but it breaks generation